query-engine/driver-adapters/js/smoke-test-js/src/libquery/pg.test.ts

```diff
@@ -1,19 +1,15 @@
 import pg from 'pg'
 import { PrismaPg } from '@jkomyno/prisma-adapter-pg'
 import { bindAdapter } from '@jkomyno/prisma-driver-adapter-utils'
+import { describe } from 'node:test'
 import { smokeTestLibquery } from './libquery'
 
-async function main() {
+describe('pg', () => {
   const connectionString = `${process.env.JS_PG_DATABASE_URL as string}`
 
-  const pool = new pg.Pool({ connectionString })
-  const adapter = new PrismaPg(pool)
+  const neonConnection = new pg.Pool({ connectionString })
+  const adapter = new PrismaPg(neonConnection)
```

Comment:
This should not actually be Neon, but just plain pG:
```suggestion
  const pool = new pg.Pool({ connectionString })
  const adapter = new PrismaPg(pool)
```

Reply:
Oops